staging/src/k8s.io/code-generator/cmd/validation-gen/validators/testing.go

```diff
@@ -123,11 +132,19 @@ func (_ fixedResultDeclarativeValidator) parseTagVal(in string) (FunctionFlags,
 		case "PtrOK":
 			flags |= PtrOK
 		default:
-			return 0, "", fmt.Errorf("unknown flag: %q", fl)
+			return tagVal{}, fmt.Errorf("unknown flag: %q", fl)
+		}
+	}
+	var typeArgs []types.Name
+	if len(pl.TypeArg) > 0 {
+		index := strings.LastIndex(pl.TypeArg, ".")
+		if index < 0 {
```

Comment:
Can the type-arg be `string` ?  Won't that fail here?

Reply:
Good catch. Fixed.